src/main/java/com/alkemy/ong/application/rest/request/AuthorizationRequest.java

```diff
@@ -0,0 +1,18 @@
+package com.alkemy.ong.application.rest.request;
+
+import javax.validation.constraints.Email;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.validator.constraints.Length;
+
+@Getter
+@Setter
+public class AuthorizationRequest {
+
+  @Email(message = "This field must be an email")
```

Comment:
'This field' en la lista de errores en el ErrorResponse no es claro o representativo del error en cuestión. Por favor, incluir el nombre del field en el mensajes: "The email has invalid format."